File: RB101-109/Easy2/ex1.rb
# Build a program that randomly generates and prints Teddy's age. 
# To get the age, you should generate a random number between 20 and 200.

# Example Output
# Teddy is 69 years old!

age = Random.new
name = 'Teddy'

loop do
  puts "Would you like to enter a name? (y/n)"
  ans = gets.chomp.downcase
  if ans.start_with?('y')
    puts "What is your name?"
    name = gets.chomp
    break
  else 
    break
  end
end
  
puts "#{name} is #{age.rand(20..200)} years old!"

File: JS210_SmallProblems/Easy3/ex1.js
// Build a program that randomly generates Teddy's age, and logs it to the
// console. Have the age be a random number between 20 and 200 (inclusive).

// Example Output:

// Teddy is 69 years old!

function randomAgeForTeddy(min, max) {
  if (min > max) [min, max] = [max, min];

  let randomNum = Number((Math.random()).toFixed(2));
  let age = Math.floor(min + (randomNum * (max - min)));
  console.log(`Teddy is ${age} years old!`);
}

randomAgeForTeddy(20, 200);